src/lib/viewers/media/DashViewer.js

```diff
@@ -426,13 +444,56 @@ class DashViewer extends VideoBaseViewer {
      * @return {void}
      */
     loadSubtitles() {
+        // Load subtitles from video, if available
         this.textTracks = this.player.getTextTracks().sort((track1, track2) => track1.id - track2.id);
         if (this.textTracks.length > 0) {
             this.mediaControls.initSubtitles(
                 this.textTracks.map((track) => getLanguageName(track.language) || track.language),
                 getLanguageName(this.options.location.locale.substring(0, 2))
             );
+            return;
```

Comment:
So the behavior for now is only try to load autogenerated captions if there are no baked in subtitles on the video?